src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -306,13 +307,13 @@ const ActionsPane = (props: {contentHeight: string}) => {
               onChange={k => setKey(k)}
               tabBarExtraContent={
                 selectedResource && resourceKindDocumentation?.helpLink ? (
-                  <Button
+                  <StyledExtraRightButton
```

Comment:
we should add a tooltip - "Open Kubernetes documentation for this resource type in external browser"?

Reply:
Yes!

Reply:
Done